src/map/label.cpp

```diff
@@ -361,7 +361,7 @@ terrain_label::terrain_label(const map_labels& parent, const config& cfg)
 terrain_label::terrain_label(terrain_label&& l)
 	: handle_(l.handle_)
 	, tooltip_handle_(l.tooltip_handle_)
-	, text_(l.text_)
+	, text_(std::move(l.text_))
```

Comment:
How come this is the only thing being moved?

Reply:
It shouldn't be. Integer parameters or other trivially copyable types shouldn't be `std::move`d, but I did miss some strings.